packages/tools/kadena-cli/src/prompts/account.ts

```diff
@@ -51,7 +55,21 @@ export const accountKdnNamePrompt: IPrompt<string> = async () =>
 export const amountPrompt: IPrompt<string> = async () =>
   await input({
     validate(value: string) {
-      return !isNaN(parseFloat(value.replace(',', '.')));
+      const parsedValue = parseFloat(value.replace(',', '.'));
+
+      if (isNaN(parsedValue)) {
+        return 'Please enter a valid amount.';
+      }
+
+      if (parsedValue <= 0) {
+        return 'Please enter a positive amount.';
+      }
+
+      if (parsedValue > 100) {
```

Comment:
Why is there a 100 limit on `amountPrompt`? This seems like a generic prompt, if there is some specific scenario where 100 limit makes sense, please name the method accordingly